### DISCUSSION TITLE
How to change checkbox label font size?

### DISCUSSION THREAD

**Discussion by barbo1942** [flet-dev/flet#1878] [Q&A] [answered]
### Question

How to change checkbox label font size?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
This is not directly possible from the Checkbox control. But you could something like this:
```py
import flet as ft

def main(page):

  page.add(
      ft.Row(
          tight=True,
          controls=[
              ft.Checkbox(),
              ft.Text("Text-made label", size=14)
          ]
      )
  )

ft.app(target=main)
```

What do you think?

**Reply by barbo1942** [reply]
Thank you so much!
